manual/python/index.md

```diff
@@ -60,9 +60,87 @@ caption="Example of graphics generated with PyROOT."
+
+### Enabling batch mode
+
+When running in batch mode, PyROOT will not display any graphics. There are two ways in which a user can activate the batch mode:
+
+1. Passing `-b` as a command line argument, e.g. `python -b my_pyroot_script.py`. For this to work, `PyConfig.IgnoreCommandLineOptions` must be set to `False` inside the PyROOT script, as discussed [here]({{ '/manual/python/#configuration-options' | relative_url }}).
+
+2. Calling `gROOT.SetBatch` in the PyROOT script, right after importing `ROOT`:
+
+```python
+import ROOT
+ROOT.gROOT.SetBatch(True)
+```
+
+### Low-level manipulation of objects
+
+When instantiating a C++ class from Python via PyROOT, both a C++ object and its Python proxy object are created. Such Python object forwards any access to its internal C++ object, thus acting as a proxy. PyROOT offers some functions to inspect or manipulate Python proxies and their C++ counterparts, based on functionality provided by [cppyy](https://cppyy.readthedocs.io/en/latest/lowlevel.html). All these functions can be accessed as `ROOT.NameOfFunction` and they are listed next:
+
+- `AddressOf(obj)`: when applied to a Proxy object `obj`, it returns an indexable buffer of length 1, whose only element is the address of the C++ object proxied by `obj`. The address of the buffer is the same as the address of the address of the C++ object. This function is kept for backwards compatibility with old PyROOT versions.
+
+- `addressof(obj, field, byref)`: similarly to `AddressOf`, it can be used to obtain the address of an internal C++ object from its Python proxy. However, `addressof` returns that address as an integer, not as an indexable buffer. Furthermore, `addressof` accepts two more parameters: `field` and `byref`. `field` can be used to specify the name of a field in a struct, in order to get the address of that field. If `byref` is set to true, `addressof` returns the address of the address of the C++ object. Equivalent to [cppyy's `addressof`](https://cppyy.readthedocs.io/en/latest/lowlevel.html#capsules). Example of usage:
+
+```python
+>>> import ROOT
+>>> ROOT.gInterpreter.Declare("struct MyStruct { int foo; float bar; };")
+True
+>>> mys = ROOT.MyStruct()
+>>> ROOT.addressof(mys) # Address of mys' C++ object
+94352024283040L
+>>> ROOT.addressof(mys, 'foo') # Address of "foo" field (same as address of object) 
+94352024283040L
+>>> ROOT.addressof(mys, 'bar') # Address of "bar" field
+94352024283044L
+>>> ROOT.addressof(mys, byref = True) # Address of address of mys' C++ object
+140376843834640L
+```
+
+Moreover, `addressof` can be used in conjunction with `TTree::Branch` from Python as shown [here](https://github.com/root-project/root/blob/d1d035e17b9b8dd97bcd146dd6e0c84a0d1aa4a1/bindings/pyroot/pythonizations/test/ttree_branch.py#L134).
```

Comment:
This link will point in some near or far future to a wrong line. Probably link only the file and describe in words where to look?

Reply:
Or just link the file, without any other comment. Should be clear enough and future safe.

Reply:
I think it won't change because if you see the link, it corresponds to a commit. It is the status of the file at that commit.